src/manifests_cert_manager.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024 Ubuntu
+# See LICENSE file for licensing details.
+
+"""Implementation of cert-manager kubernetes manifests."""
+
+import logging
+import pickle
+from hashlib import md5
+from typing import Dict
+
+from ops.manifests import ConfigRegistry, ManifestLabel, Manifests, Patch
+
+log = logging.getLogger()
+
+
+class RemoveNamespace(Patch):
+    """Remove namespace from resources so they deploy to the charm's namespace."""
+
+    def __call__(self, obj):
+        """Remove namespace from resources."""
+        obj.metadata.namespace = None
+
+
+class CertManagerManifests(Manifests):
```

Comment:
It looks to me like this `CertManagerManifests` doesn't really do much altering?  I don't even see an `upstream/cert-manager` folder with changes.  I'm guessing they're already in the repo??

Reply:
Correct, those are already in the repo from a previous PR.